content/en/docs/concepts/security/security-checklist.md

```diff
@@ -206,6 +206,24 @@ SELinux is only available on Linux nodes, and enabled in
 [some Linux distributions](https://en.wikipedia.org/wiki/Security-Enhanced_Linux#Implementations).
 {{< /note >}}
 
+## Logs and auditing
+
+- [ ] Audit logs, if enabled, are protected from general access.
+- [ ] The `/logs` API is disabled (you are running kube-apiserver with
+  `--enable-logs-handler=false`).
+
+  Kubernetes includes a deprecated `/logs` API endpoint, enabled by default,
+  that lets users request API server logs over HTTP. Accessing that endpoint
+  requires authentication.
+
+Allowing broad access to Kubernetes logs can make security information
+available to a potential attacker.
+
+As a good practice, set up a separate means to collect and aggregate
+control plane logs, and do not use the deprecated `/logs` API endpoint.
+Alternatively, if you run your control plane with the `/logs` API endpoint
+enabled,  configure authorization rules that restrict access appropriately.
```

Comment:
```suggestion
enabled, configure authorization rules that restrict access appropriately,
and limit the content of /var/log to Kubernetes API server logs.
```

Reply:
yeah, though maybe make it singular "within the host or container" to match the singular "where the API server is running"?